src/main/java/org/humancellatlas/ingest/state/SubmissionState.java

```diff
@@ -77,7 +82,7 @@ public boolean after(SubmissionState submissionState) {
 
     public static SubmissionState fromString(String submissionState) throws UnrecognisedSubmissionStateException {
         try {
-            return SubmissionState.valueOf(submissionState.toUpperCase());
+            return SubmissionState.valueOf(submissionState.replaceAll(" ", "_").toUpperCase());
```

Comment:
Consider extracting this string conversion code to a function and reusing in all places that use it, e.g. `SubmissionStateUpdater.java`

Reply:
I have done it a bit differently but to the same effect. I think it's a bit neater 938a92ec6b623bf109b86eb2359a683f5efa8edb